packages/ui/src/assets/ShoppingCart.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import type { FC } from 'react'
+
+// Icon from Phosphor Icons
+const ShoppingCart: FC = () => (
```

Comment:
Added this "temporary" folder with `ShoppingCart` icon that was removed [here](https://github.com/vtex/faststore/pull/1501/files) but still used in other components' stories.